### PR TITLE
Added info service and tests

### PR DESCRIPTION
**Pull request by jlavendtt**
Added info repo
Added info service Layer
Added Info tests

### CODE REVIEW

**Review by neveland98 — APPROVED** [review]
Looks good. Thank you for testing your code before submitting it. :)

File: src/main/java/com/talentpath/shamazin/showItemPage/services/InfoService.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    public void deleteInfo(Integer id) {
+        infoDao.deleteById(id);
+    }
+
+    public Info getInfo(Integer id) throws NoSuchItemException {
+        Optional<Info> info = infoDao.findById(id);
+        if(info.isPresent()) return info.get();
+        else throw new NoSuchItemException("No item with id: " + id);
```
Comment (timluu) [inline src/main/java/com/talentpath/shamazin/showItemPage/services/InfoService.java:35]: Do we have cases where an item is present with no info description? If so, NoSuchItemException and this message might not give an accurate representation of that error. If not, then we can just ignore my feedback

File: src/test/java/com/talentpath/shamazin/showItemPage/services/ItemServiceTest.java [hunk 2]
```diff
@@ -70,6 +70,10 @@ void addItem() {
         }
 
     }
+    @Test
+    void practiceTest() {
+
+    }
```
Comment (timluu) [inline src/test/java/com/talentpath/shamazin/showItemPage/services/ItemServiceTest.java:76]: What is this test for?

**Review by timluu — CHANGES_REQUESTED** [review]
One comment regarding the NoSuchItemException. If we ignore that though, everything else LGTM 👍🏻

**Review by timluu — APPROVED** [review]
lgtm

**Pull request merged**